test/framework/kogitojobsservice.go

```diff
@@ -16,135 +16,82 @@ package framework
 
 import (
 	"fmt"
-	"strconv"
 
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
 	"github.com/kiegroup/kogito-cloud-operator/test/config"
-	apps "k8s.io/api/apps/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
-// InstallKogitoJobsService deploy the Kogito Jobs service
+// InstallKogitoJobsService returns the installation service for the Kogito Jobs Service
```

Comment:
This should be changed. it does not return

Reply:
Changed